common-lisp.html.markdown

```diff
@@ -4,82 +4,91 @@ language: "Common Lisp"
 filename: commonlisp.lisp
 contributors:
   - ["Paul Nathan", "https://github.com/pnathan"]
+  - ["Rommel Martinez", "https://ebzzry.io"]
 ---
 
-ANSI Common Lisp is a general purpose, multi-paradigm programming
```

Comment:
Can you explain your removal of ANSI Common Lisp? I believe it is still in ANSI standard.

Reply:
Yes.

ANSI Common Lisp is the standard that was published in 1994. Since then, the Common Lisp that we use now has undergone several changes. An important aspect of it is how the implementations decided to add features to the base ANSI CL—features like concurrency, Unicode, CLOS, etc.

When one speaks of ANSI Common Lisp now, they are referring to that standard. The term “Common Lisp,” on the other hand refers to the general version that are currently in use now, as how it has evolved since then.

I hope that helps. :)

Reply:
Yeah. You're right. I do common lisp. I didn't know about this.

Reply:
So, the current Common Lisp is a fork of ANSI CL?

Reply:
Yes, in spirit. More like, the current Common Lisp is a superset-ish of ANSI CL. :)